classes/Combination.php

```diff
@@ -79,7 +79,7 @@ class CombinationCore extends ObjectModel
         'primary' => 'id_product_attribute',
         'fields' => [
             'id_product' => ['type' => self::TYPE_INT, 'shop' => 'both', 'validate' => 'isUnsignedId', 'required' => true],
-            'location' => ['type' => self::TYPE_STRING, 'validate' => 'isGenericName', 'size' => 64],
+            'location' => ['type' => self::TYPE_STRING, 'validate' => 'isString', 'size' => 255],
```

Comment:
```suggestion
            'location' => ['type' => self::TYPE_STRING, 'validate' => 'isString', 'size' => 255, 'allow_null' => true],
```

Reply:
Please update phpdoc for attribute

Reply:
removed `location` from definition. Its read-only now in combination and is loaded from stockAvailable (homogenized with product)